xayn-ai/src/coi/point.rs

```diff
@@ -6,6 +6,50 @@ use serde::{Deserialize, Serialize};
 
 use crate::{coi::CoiId, embedding::utils::Embedding, utils::system_time_now};
 
+#[derive(Clone, Copy, Deserialize, Serialize)]
+#[cfg_attr(test, derive(Debug))]
+pub(crate) struct CoiView {
```

Comment:
i agree, the name coincidence is unfortunate. i renamed it to `CoiStats` and prefixed/suffixed the fields, since stats can contain more than just view stats. also renamed the getter/setter in `CoiPoint`.